assets/components/src/style.scss

```diff
@@ -0,0 +1,44 @@
+@import '~@wordpress/base-styles/colors';
+@import '../shared/scss/colors';
```

Comment:
I was also uncertain whether importing assets from outside the `components` folder would cause issues when `newspack-components` is imported as an NPM package, but I tested with importing using `file:../newspack-plugin/assets/components` from another repo and didn't encounter any errors. 

Probably because the assets must be compiled before publishing, but good to confirm!

Reply:
This is handled in the pre-publish step:

https://github.com/Automattic/newspack-plugin/blob/9dbff69ebee3dd090f9ee19e414b128c8e575ed4/assets/components/package.json#L50

The files from outside of `components` are copied there.